### DISCUSSION TITLE
Duplicate entry 'local' for key 'PRIMARY'

### DISCUSSION THREAD

**Discussion by MaximalMax07** [requarks/wiki#6606] [Error / Bug Report]
insert into `authentication` (`autoEnrollGroups`, `config`, `displayName`, `domainWhitelist`, `isEnabled`, `key`, `order`, `selfRegistration`, `strategyKey`) values ('{\"v\":[]}', '{}', 'Local', '{\"v\":[]}', true, 'local', 0, false, 'local') - Duplicate entry 'local' for key 'PRIMARY'

getting this error when trying to create the Admin Account on the installation screen
![grafik](https://github.com/requarks/wiki/assets/82326560/c5f12448-399a-4a71-893c-4a9bed9cc31c)

This error comes to me sometimes too 
![grafik](https://github.com/requarks/wiki/assets/82326560/e321713f-7cb4-4074-bac1-cb4888c36970)

**Comment by NGPixel** [comment]
Provide your database type and version

**Reply by MaximalMax07** [reply]
i used mariadb. 
i already fixed it myself by reinstalling wikijs 
sorry for late reply
